### PR TITLE
Change readme syntax

### PR DESCRIPTION
**Pull request by zachneill**
This PR makes modifications. The changes are minor, such as word misspells or grammatical fixes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,12 @@
 [**Audacity**](https://www.audacityteam.org) is an easy-to-use, multi-track audio editor and recorder for Windows, macOS, GNU/Linux and other operating systems. Audacity is open source software licensed under GPL, version 2 or later.
 
 - **Recording** from any real or virtual audio device that is available to the host system.
-- **Export / Import** a wide range of audio formats, extendible with FFmpeg.
+- **Export / Import** a wide range of audio formats, extendable with FFmpeg.
```
Comment (Paul-Licameli) [inline README.md:7]: Should that be "extensible"?

Comment (zachneill) [reply README.md:7]: I think they are pretty much synonymous. Extensible is probably better though. I'll change it!

Comment (zachneill) [reply README.md]: Changed!

**Review by LWinterberg — APPROVED** [review]
Look, Paul, you even get a few oxford commas back :)

**Pull request merged**